test/test_ops.py

```diff
@@ -81,6 +81,7 @@ def test_ones_like(self):
   def test_eye(self):
     helper_test_op([], lambda: torch.eye(10), lambda: Tensor.eye(10), forward_only=True)
 
+  @unittest.skipIf(IMAGE>0, "no arange on images")
```

Comment:
Why? I want single dim cumsum to work on images.

Reply:
Fair enough. Cumsum works on images, it's arange that fails because arange asks for cumsum of a 1-d tensor. My implementation of cumsum relies on conv2d taking arbitrary many dimensions, however image_conv2d can only take 2d kernels.

Can see obvious options as:

Option 1: rewrite cumsum to only call conv2d with a true 2d kernel. This should take +1 or +2 lines of code, requiring some permutes before and after the 1d cumsum implementation.

Option 2 (hackier): Make arange call cumsum on a 2d tensor with shape (1,n) and flatten after. No extra lines of code, and arange now works on images.

Reply:
Option 1 sounds good, I'm okay with a few lines.

Reply:
This seems to work nicely now.